tools/db/replicateDbs.py

```diff
@@ -79,6 +79,11 @@ def isExcluded(dbName):
         filterDesignDocument = sourceDb[db].get("_design/%s" % filterName)
         if not args.continuous and filterDesignDocument:
             replicateDesignDocument["filter"] = "%s/withoutDeletedAndDesignDocuments" % filterName
+
+        existingDoc = replicator.get(backupDb)
```

Comment:
My suggestion would be not to update the script because of the test failures.
As this is a script that could be used in production environments, my opinion is, that the script should fail if the replication already exists (maybe with different values) otherwise something could be changed unexpectedly.

Reply:
@cbickel, sounds good. I'll revert changes to the Python script.